### PR TITLE
Add prerequisites section to configure the GitLab App

### PR DESCRIPTION
**Pull request by praneetloke**
Since all new organizations are Pulumi-managed, by default, if a user wants to configure the GitLab app, they must ensure a couple of things before doing the GitLab side of the setup. This PR clarifies those prerequisites.

Page to review: Under the **User Guides** nav header > Continuous Delivery > Pulumi GitLab Integration

### CODE REVIEW

File: themes/default/content/docs/guides/continuous-delivery/gitlab-app.md [hunk 1]
```diff
@@ -16,10 +16,17 @@ To enable the integration with your GitLab project, you will need to ensure you
 * [Signup](https://app.pulumi.com/signup) for a Pulumi account with your GitLab identity (or link your GitLab identity with an existing account.)
 * If your GitLab Project is under a GitLab Group, ensure that the group is added to Pulumi as an [organization](https://app.pulumi.com/site/trial)
 
+## Prerequisites
+
+* You must have a GitLab identity associated with your Pulumi account
```
Comment (cnunciato) [inline themes/default/content/docs/guides/continuous-delivery/gitlab-app.md:21]:
```suggestion
* You must have a GitLab identity associated with your Pulumi account.
```

File: themes/default/content/docs/guides/continuous-delivery/gitlab-app.md [hunk 2]
```diff
@@ -16,10 +16,17 @@ To enable the integration with your GitLab project, you will need to ensure you
...
+* If you are integrating a GitLab project that is under a group, [add an organization]({{< relref "/docs/intro/console/organizations#creating-an-organization" >}})
```
Comment (cnunciato) [inline themes/default/content/docs/guides/continuous-delivery/gitlab-app.md:22]:
```suggestion
* If you are integrating a GitLab project that is under a group, [add an organization]({{< relref "/docs/intro/console/organizations#creating-an-organization" >}}).
```

File: themes/default/content/docs/guides/continuous-delivery/gitlab-app.md [hunk 3]
```diff
@@ -16,10 +16,17 @@ To enable the integration with your GitLab project, you will need to ensure you
...
+  in Pulumi.
+  * After you add the organization ensure that it uses GitLab as its identity provider. You may [change your org identity]({{< relref "/docs/intro/console/organizations#changing-identity-providers" >}})
+    provider in the Pulumi Console after the organization has been added.
```
Comment (cnunciato) [inline themes/default/content/docs/guides/continuous-delivery/gitlab-app.md:25]:
```suggestion
  * After you add the organization, ensure that it uses GitLab as its identity provider. You may [change your org identity provider]({{< relref "/docs/intro/console/organizations#changing-identity-providers" >}}) in the Pulumi Console under Settings > Access Management > Membership Requirements.
```

**Review by cnunciato — APPROVED** [review]
A few little suggestions, otherwise 👍 -- thanks!

File: themes/default/content/docs/guides/continuous-delivery/gitlab-app.md (hunk 3 above)
Comment (susanev) [reply themes/default/content/docs/guides/continuous-delivery/gitlab-app.md:25]: seems a bit weird to link to the content that describes that as well as describing it here? i think we should prolly choose one or the other.

Comment (cnunciato) [reply themes/default/content/docs/guides/continuous-delivery/gitlab-app.md:25]: Ah, fair point -- maybe something like this, then:

```suggestion
  * After you add the organization [ensure that it uses GitLab as its identity provider]({{< relref "/docs/intro/console/organizations#changing-identity-providers" >}}). 
```

File: themes/default/content/docs/guides/continuous-delivery/gitlab-app.md [hunk 4]
```diff
@@ -16,10 +16,16 @@ To enable the integration with your GitLab project, you will need to ensure you
 * [Signup](https://app.pulumi.com/signup) for a Pulumi account with your GitLab identity (or link your GitLab identity with an existing account.)
 * If your GitLab Project is under a GitLab Group, ensure that the group is added to Pulumi as an [organization](https://app.pulumi.com/site/trial)
 
+## Prerequisites
+
+* You must have a GitLab identity associated with your Pulumi account.
+* If you are integrating a GitLab project that is under a group, [add an organization]({{< relref "/docs/intro/console/organizations#creating-an-organization" >}}).
```
Comment (praneetloke) [inline themes/default/content/docs/guides/continuous-delivery/gitlab-app.md:22]:
```suggestion
* If you are integrating a GitLab project that is under a group, [add an organization]({{< relref "/docs/intro/console/organizations#creating-an-organization" >}})
```

Comment (praneetloke) [reply themes/default/content/docs/guides/continuous-delivery/gitlab-app.md:22]: Looks like the line-break deceived us into thinking this was the end of the line

**Pull request merged**